src/components/pages/room.js

```diff
@@ -433,25 +439,25 @@ export default function Room({ PageTemplate, initialState }) {
                 label: t('settingsTabTitle'),
                 children: (
                   <div className="Tabs-tabPane" >
-                    <Card className="RoomPage-card" title={t('updateRoomCardTitle')}>
+                    <div className="RoomPage-sectionHeadline">{t('roomMetadataHeadline')}</div>
+                    <section className="RoomPage-metadataSection">
```

Comment:
As this is the only section in the tab I would suggest that we do not need a title and additional indentation for the section content. Watcha think?

Reply:
Edit: I realized this is because it makes the effect of the update button clearer. So please ignore the comment.

Reply:
I actually had removed the title cause I had the same thought, but then I realized we might need to adjust all labels, because it seemed a bit vague to have just a "Name" input, given we don't (yet 🤞 ) have a way to clearly showing the user the context they are in.. But let's see how this evolves.